powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/LoggingUtils.java

```diff
@@ -93,15 +48,6 @@ public static String getCorrelationId() {
         return MDC.get(CORRELATION_ID.getName());
```

Comment:
what should we do with this one? that's the only one remaining ? and the objectMapper. 
I think we should remove the correlation ID, and document the usage of MDC (correlationIdPath is still here in the annotation).
Regarding the ObjectMapper, not sure it is useful, they can also use JsonUtils from Serialization module.

We should completely remove the LoggingUtils, WDYT?